app/views/user_checklists/show_progress.html.haml

```diff
@@ -67,8 +65,9 @@
 
                   - next_sibling = @user_checklist.next_sibling
                   - if next_sibling
-                    .next
-                      = link_to  user_user_checklist_progress_path(@user, next_sibling), { id: next_link_id, class: hide_class } do
+                    -# start with this not displayed; javascript below will display it via onchange
+                    .next{ id: next_link_id, style: 'display: none;' }
```

Comment:
Can we use class instead? (`d-none` or `hide_class`)

Reply:
There was some weirdness with using a class vs. setting the display to none; setting the display to 'none' always worked, but using a class sometimes didn't work due to something having to do with when the DOM was loaded and updated.  (javascript + testing).  This has been working working reliably so I'd like to keep it like this for now.

But I'll add in a comment to the code so that it can be revisited.